tests/framework/extensions/clusters/clusters.go

```diff
@@ -68,6 +69,27 @@ func IsHostedProvisioningClusterReady(event watch.Event) (ready bool, err error)
 	return false, nil
 }
 
+// Verify if a serviceAccountTokenSecret exists or not in the cluster.
+func CheckServiceAccountTokenSecret(client *rancher.Client, clusterName string) (success bool, err error) {
+	clusterID, err := GetClusterIDByName(client, clusterName)
+	if err != nil {
+		return false, err
+	}
+
+	cluster, err := client.Management.Cluster.ByID(clusterID)
+	if err != nil {
+		return false, err
+	}
+
+	if cluster.ServiceAccountTokenSecret != "" {
+		logrus.Info("serviceAccountTokenSecret in this cluster is: %s\n", cluster.ServiceAccountTokenSecret)
```

Comment:
use `Infof` Markus, so you can pass the string like how you are doing now, and also you don't need "\n"

Reply:
Sure thing, thanks!! Oh okay, I had the `\n` just so that I would push the `===CONT` to the next line but I can remove it

Reply:
Good to go now. That's a neat package! Was unaware of it....